CMakeLists.txt

```diff
@@ -138,3 +135,10 @@ enable_testing()
 add_test(NinjaTest ninja_test)
 
 install(TARGETS ninja DESTINATION bin)
+
+include(FeatureSummary)
+
+add_feature_info(LTO lto_supported "Link time / interprocess optimization  ${error}")
```

Comment:
${error} doesn't seem to work now?

Reply:
I think ${error} prints as expected; if I manually do

```cmake
set(error "hello")
```

and then

```sh
cmake -B build
```

```
-- The following features have been disabled:

 * LTO, Link time / interprocess optimization  hello
 * re2c, lexer generator
```

Some compilers that don't have LTO like PGI also don't have any "error" output